tests/exchanges/connectors/test_ccxt_exchange.py

```diff
@@ -129,36 +130,41 @@ async def test_get_ccxt_order_type(exchange_manager):
 
 
 async def test_get_trade_fee(exchange_manager, future_trader_simulator_with_default_linear):
+    future_symbol = "BTC/USDT:USDT"
+    future_fees_value = 0.0004
+    spot_symbol = "BTC/USDT"
     config, fut_exchange_manager_inst, trader_inst, default_contract = future_trader_simulator_with_default_linear
     fut_exchange_manager_inst.is_future = True
     fut_ccxt_exchange = exchange_connectors.CCXTExchange(config, fut_exchange_manager_inst)
     spot_ccxt_exchange = exchange_connectors.CCXTExchange(exchange_manager.config, exchange_manager)
     # spot trading
     spot_ccxt_exchange.client.options['defaultType'] = enums.ExchangeTypes.SPOT.value
     await spot_ccxt_exchange.client.load_markets()
-    assert spot_ccxt_exchange.get_trade_fee("BTC/USDT", enums.TraderOrderType.BUY_LIMIT, decimal.Decimal("0.45"),
+    assert spot_ccxt_exchange.get_trade_fee(spot_symbol, enums.TraderOrderType.BUY_LIMIT, decimal.Decimal("0.45"),
                                             decimal.Decimal(10000), "taker") == \
            _get_fees("taker", "BTC", 0.001, decimal.Decimal("0.00045"))
-    assert spot_ccxt_exchange.get_trade_fee("BTC/USDT", enums.TraderOrderType.SELL_LIMIT, decimal.Decimal("0.45"),
+    assert spot_ccxt_exchange.get_trade_fee(spot_symbol, enums.TraderOrderType.SELL_LIMIT, decimal.Decimal("0.45"),
                                             decimal.Decimal(10000), "maker") == \
-           _get_fees("maker", "USDT", 0.001, decimal.Decimal("10"))
+           _get_fees("maker", "USDT", 0.001, decimal.Decimal("4.5"))
     # future trading
     fut_ccxt_exchange.client.options['defaultType'] = enums.ExchangeTypes.FUTURE.value
     await fut_ccxt_exchange.client.load_markets()
+    # enforce taker and maker values
+    set_future_exchange_fees(fut_ccxt_exchange, future_symbol, taker=future_fees_value, maker=future_fees_value)
```

Comment:
👍 
I think we could also add one more thing here before this overrite to make sure our mock has the right fee order of magnitude. I mean to make sure that fees are still fractions and not %.
Something like:
`assert future_fees_value/5 <= fut_ccxt_exchange.client.markets[symbol]['taker']  <= future_fees_value*5`
and do the same for spot fees.
This way if for some reason ccxt switches fees values from a fraction to a %, we will know it (it happened before)